src/marked.js

```diff
@@ -0,0 +1,66 @@
+import { customHeadingRenderer } from './markdown_renderers';
+
+const matchAll = require('match-all');
+
+const marked = require('marked');
+
+export const apifyMarked = (() => {
+    const renderer = new marked.Renderer();
+    renderer.heading = customHeadingRenderer;
+    renderer.code = (code) => {
+        if (!code.trim().startsWith('```marked-tabs')) {
+            return false;
+        }
+
+        const matchesIterator = matchAll(code, /<marked-tab header="(?<header>.*?)" lang="(?<lang>.*?)">(?<code>.*?)<\/marked-tab>/sg);
```

Comment:
Food for thought: I am just thinking that usage of comments could be more readable when viewing and editing source markdown files at Github https://monosnap.com/file/rLXwI5FnrNVfqlPMtSoCL2K3cmmtdV but I don't have a strong opinion on this :)

Reply:
I've tried it out and I don't see a clear advantage of comments. MD editors (stackedit online and bear offline) seem not to highlight comments. Also, comments has disadvantage that they don't have closing `>` tag (or some other closing tag) which is worse for matching. So I'd stick with `<>`.